<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFlatUI</class>
 <widget class="QWidget" name="frmFlatUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QRadioButton" name="rbtn1">
     <property name="text">
      <string>语文</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btn3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>测试按钮</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="txt3"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btn2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>测试按钮</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="9">
    <widget class="QSlider" name="slider3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QRadioButton" name="rbtn2">
     <property name="text">
      <string>英语</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QProgressBar" name="bar2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QScrollBar" name="horizontalScrollBar">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QRadioButton" name="rbtn4">
     <property name="text">
      <string>历史</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QRadioButton" name="rbtn3">
     <property name="text">
      <string>数学</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btn4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>测试按钮</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txt2"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="txt4"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btn1">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>测试按钮</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="txt1"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="bar1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="9">
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QSlider" name="slider1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QSlider" name="slider2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashDotLine</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
